osu.Game/Overlays/Profile/Sections/Ranks/DrawableTotalScore.cs

```diff
@@ -2,30 +2,260 @@
 // See the LICENCE file in the repository root for full licence text.
 
 using osu.Framework.Allocation;
+using osu.Framework.Extensions.Color4Extensions;
 using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Localisation;
+using osu.Game.Beatmaps;
 using osu.Game.Graphics;
+using osu.Game.Graphics.Containers;
 using osu.Game.Graphics.Sprites;
+using osu.Game.Online.Leaderboards;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.UI;
 using osu.Game.Scoring;
+using osuTK;
+using osuTK.Graphics;
+using System.Collections.Generic;
 
 namespace osu.Game.Overlays.Profile.Sections.Ranks
 {
-    public class DrawableTotalScore : DrawableProfileScore
+    public class DrawableTotalScore : OsuHoverContainer
     {
+        private const int height = 40;
+        private const int corner_radius = 6;
+
+        protected readonly Container InfoContainer;
+        protected readonly FillFlowContainer Accuracy;
```

Comment:
This should be called something else. You are using it to contain more than just accuracy.